src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -698,7 +717,7 @@ def forward(
         >>> # model predicts one of the 400 Kinetics-400 classes
         >>> predicted_label = logits.argmax(-1).item()
         >>> print(model.config.id2label[predicted_label])
-        eating spaghetti
+        LABEL_116
```

Comment:
The model actually doesn't output `eating spaghetti` it outputs `LABEL_116`which corresponds to `eating spaghetti` in kinetics 400.